worker/jobTypes/githubPushJob.js

```diff
@@ -146,13 +149,15 @@ async function cloneRepo(currentJob) {
   }
   // clone the repo we need to build
   try {
-    let repoPath =
-      'https://github.com/' +
-      currentJob.payload.repoOwner +
-      '/' +
-      currentJob.payload.repoName;
 
-    console.log(repoPath);
+    let basePath = `https://github.com`;
+    
+    // private repos use builder bot from github to pull
+    if (currentJob.payload.private) {
+      basePath = `https://${process.env.GITHUB_BOT_USERNAME}:${process.env.GITHUB_BOT_PASSWORD}@github.com`;
```

Comment:
Since we are repeating this code, I'd just extract it to a function called getBasePath(currentJob.payload) returning the basePath.